CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ permalink: /docs/en-US/changelog/
 
 # Changelog
 
-## 3.15 ( 2025 May )
+## 3.15.1 ( 2025 May 21st )
+
+This release brings with it an update to Ubuntu 24, it's recommended to upgrade for continued support. To do this backup your database, destroy your VM, reprovision, and restore the database. A `vagrant ssh -c db_restore` script is available inside the VM to perform these backups, and will place SQL dumps in the `database/sql/backups` folder if backups and restores are enabled in your `config/config.yml`. There is also a `vagrant ssh -c db_restore` command for importing those backups.
```

Comment:
VVV already do backups

Reply:
Not everyone has it turned on, running the script ensures it always happens wether it's automated or not